src/commands.ts

```diff
@@ -760,6 +762,175 @@ function humanReadable(record: HistoryRecord, denominator: number): string {
+        //  - the nullifier doesn't take into account (it's not belong to us)
+        //  - the next nullifier cannot be calculated without output account
+        if (aRecord.type != HistoryTransactionType.TransferIn) {
+            this.echo(`\tNullifier: ${bufToHex(aRecord.nullifier)}`);
+            if (aRecord.nextNullifier) {
+                this.echo(`\tNext nullifier: ${bufToHex(aRecord.nextNullifier)}`);
+            }
+
+            this.echo(`\tAccount @${aRecord.index}: ${JSON.stringify(aRecord.acc)}`);
+
+            let accEnc = findChunk(aRecord.index, aRecord.encChunks);
+            let accKey = findKey(aRecord.index, aRecord.ecdhKeys);
+            if (accEnc && accKey) {
+                this.echo(`\t      encrypted: ${bufToHex(accEnc)}`);
+                this.echo(`\t      ECDH key:  ${bufToHex(accKey)}`);
+            } else {
+                this.echo(`[[;red;]Incorrect report: cannot find compliance details for account @${aRecord.index}]`);
+            }
+        }
+        
+        for (const aNote of aRecord.notes) {
+            this.echo(`\tNote    @${aNote.index}: ${JSON.stringify(aNote.note)}`);
+
+            let noteEnc = findChunk(aNote.index, aRecord.encChunks);
+            let noteKey = findKey(aNote.index, aRecord.ecdhKeys);
+            if (noteEnc && noteKey) {
+                this.echo(`\t      encrypted: ${bufToHex(noteEnc)}`);
+                this.echo(`\t      ECDH key:  ${bufToHex(noteKey)}`);
+            } else {
+                this.echo(`[[;red;]Incorrect report: cannot find compliance details for note @${aNote.index}]`);
+            }
+        }
+
+        const inputs = aRecord.inputs;
+        if (inputs) {
+            this.echo(`\t[[;green;]Input account @${inputs.account.index}:] ${JSON.stringify(inputs.account.account)}`);
+            this.echo(`\t\t...intermediate nullifier hash: ${inputs.intermediateNullifier}`);
+            for (const aNote of inputs.notes) {
+                this.echo(`\t[[;green;]Input note    @${aNote.index}:] ${JSON.stringify(aNote.note)}`);
+            }
+        }
+    }
+
+    this.echo('[[;white;]--------------------------------END-OF-REPORT--------------------------------]\n');
+
+    let metadata: any = {};
+    metadata.userId = this.account.getAccountId();
+    metadata.startTimestamp = fromDate ? fromDate.getTime() : null;
+    metadata.endTimestamp = toDate ? toDate.getTime() : null;
+    metadata.recordsCount = report.length;
+
+    const space = 4;
+    const replacer = (key, value) => {
+        if (typeof value === 'bigint') {
+            return value.toString() + 'n';
+        } else if (value instanceof Object.getPrototypeOf(Uint8Array)) {
+            return `${bufToHex(value)}`;
+        }  else {
+            return value;
+        }
+    }
+
+    let zipFile = new JSZip();
+    zipFile.file(`metadata.json`, JSON.stringify(metadata, replacer, space));
+    zipFile.file(`transactions.json`, JSON.stringify(report, replacer, space));
```

Comment:
Why do we need 2 separate files?

Reply:
Actually it's just an export example. The compliance report design will be formed by application. So I think it doesn't matter. Currently I rewritten it to export a single file at https://github.com/zkBob/zkbob-console/commit/e29de03e3d1f965765e97594c5a5ec4a5cde866b
A little benefit of two files: macOS unpack every report in the separate folders :)

New report format: 
[26939d7e-e30f-4e3c-87a4-7eefa028239d.zip](https://github.com/zkBob/zkbob-console/files/10527685/26939d7e-e30f-4e3c-87a4-7eefa028239d.zip)